src/store.ts

```diff
@@ -238,63 +238,6 @@ export class DiscordStore {
             throw err;
         }
     }
-
-    public async get_dm_room(discordId, discordChannel): Promise<string> {
```

Comment:
meow? huge deleted blocks?

Reply:
Yeah we don't use them for anything. Even the DM branch doesn't use them :D